packages/donations-app/app/components/project-wizard/projectsView.html

```diff
@@ -34,7 +34,7 @@ <h2 class="section" style="margin-top: 0px;">Projects</h2>
           Make active
         </a>
         <a href="/project-wizard/{{project.code}}" class="btn btn-warning" >Edit</a>
-        <a href="/project/{{project.code}}/" class="btn btn-success">Preview</a>
+        <a href="/project/{{project.code}}/{{project.appealPageVersion}}" class="btn btn-success">Preview</a>
```

Comment:
It's just an idea ;)
Have you considered having a default project view (a link without a version) and settling down the appealPage version in the router in a single place instead of repeating the login in every url?

Reply:
I see, it's a great idea